hdtv/options.py

```diff
@@ -25,15 +25,19 @@
 
 from html import escape
 
+
 class Option(object):
     """
     A configuration variable
     """
 
-    def __init__(self, default=None,
-                 parse=lambda x: x,
-                 toStr=lambda x: str(x),
-                 changeCallback=None):
+    def __init__(
```

Comment:
Codacy found an issue: [Missing docstring in __init__ (D107)](https://app.codacy.com/manual/janmayer/hdtv/pullRequest?prid=6293112)